databases/core.py

```diff
@@ -489,3 +531,20 @@ def __repr__(self) -> str:
 
     def __eq__(self, other: typing.Any) -> bool:
         return str(self) == str(other)
+
+
+class FakeExecutionContext:
+    """
+    This is an object that raises an error when one of its properties are
+    attempted to be accessed. Because we're not _really_ using SQLAlchemy
+    (besides using its query builder), we can't pass a real ExecutionContext
+    to ColumnDefault objects. This class makes it so that any attempts to
+    access the execution context argument by a column default callable
+    blows up loudly and clearly.
+    """
+
+    def __getattr__(self, _: str) -> typing.NoReturn:  # pragma: no cover
+        raise NotImplementedError(
```

Comment:
btw: my own custom implementation of this (yep, I have a similar hack in my prod), I pass the current `values` to this context so that it becomes much more useful.